### DISCUSSION TITLE
Lithium + Ketamine

### DISCUSSION THREAD

**Discussion by zelixir25** [TripSit/drugs#108] [Combination Chart]
I found a few studies looking at this combination for medical purposes. Diazgranados, et al. (2010) administered ketamine to patients taking lithium and did not report any significant adverse effects aside from dissociative effects, which is expected from ketamine. Scheuing, et al. (2015) found that, in clinical settings, lithium may enhance and prolong the antidepressant effects of ketamine. They also did not report any serious adverse effects nor did the lithium seem to prolong the acute dissociative effects.

Diazgranados, N., Ibrahim, L., Brutsche, N. E., Newberg, A., Kronstein, P., Khalife, S., Kammerer, W. A., Quezado, Z., Luckenbaugh, D. A., Salvadore, G., Machado-Vieira, R., Manji, H. K., & Zarate, C. A., Jr. (2010). A randomized add-on trial of an N-methyl-D-aspartate Antagonist in treatment-resistant bipolar depression. Archives of General Psychiatry, 67(8), 793–802. https://doi.org/10.1001/archgenpsychiatry.2010.90 

Scheuing, L., Chiu, C.-T., Liao, H.-M., & Chuang, D.-M. (2015). Antidepressant mechanism of ketamine: perspective from preclinical studies. Frontiers in Neuroscience, 9. https://doi.org/10.3389/fnins.2015.00249

**Comment by utaninja** [comment]
Wanted to link this that discusses the interaction- https://www.dovepress.com/ketamine-in-bipolar-disorder-a-review-peer-reviewed-fulltext-article-NDT

**Comment by utaninja** [comment]
Pushed to https://github.com/TripSit/drugs/issues/301
